### PR TITLE
feat(pkgs/dmd): Version 2.096.1 supported

### PR DESCRIPTION
**Pull request by dukc**
Not that this does much currently, but at least it's a placeholder that we remember this version works.

### CODE REVIEW

**Comment by dukc** [comment]
Ah well, time to move this to the supported versions json file instead. Doing it sometime later, trying to find time for it this week.

**Comment by dukc** [comment]
@PetarKirov moved!

This is starting to be in such a good shape that I'm soon going to archive oldDDerivations. Especially the huge wealth of binary packages to pick from here is quite impressive!

**Comment by PetarKirov** [comment]
> @PetarKirov moved!

Hey @dukc, thanks, good work 👍. The `dmd-2_096_1 | x86_64-linux` job is all good, however the `x86_64-darwin` counterpart is failing. I've pushed a commit to disable the failing tests on dmd's test suite, however now the core.thread.fiber unit tests are segfaulting. To disable them I would probably need to include a patch specific to this version. An alternative solution is to disable the checkPhase entirely. I'll see what I can do. Considering that the same test work on `x86_64-darwin` works fine with 2.098.1, it maybe the most practical, choice though I'd prefer to avoid it.

> This is starting to be in such a good shape that I'm soon going to archive oldDDerivations. Especially the huge wealth of binary packages to pick from here is quite impressive!

I'm very happy to hear that!

File: pkgs/dmd/build-status.json [hunk 1]
```diff
@@ -1,4 +1,15 @@
 {
+  "2.096.1": {
+    "x86_64-darwin": {
+      "build": true,
+      "check": true,
```
Comment (PetarKirov) [inline pkgs/dmd/build-status.json:5]:
```suggestion
      "check": false,
```

**Pull request merged**